receiver/zipkinreceiver/config.go

```diff
@@ -4,22 +4,89 @@
 package zipkinreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver"
 
 import (
+	"fmt"
+
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/config/confighttp"
+	"go.opentelemetry.io/collector/confmap"
+	"go.opentelemetry.io/collector/featuregate"
+)
+
+var disallowHTTPDefaultProtocol = featuregate.GlobalRegistry().MustRegister(
+	"zipkinreceiver.httpDefaultProtocol.disallow",
+	featuregate.StageAlpha,
+	featuregate.WithRegisterDescription("When enabled, usage of the default http configuration is disallowed"),
+	featuregate.WithRegisterFromVersion("v0.114.0"),
```

Comment:
I'm sorry this has been taking so long :sweat: 
```suggestion
	featuregate.WithRegisterFromVersion("v0.122.0"),
```